components/scream/src/physics/spa/atmosphere_prescribed_aerosol_hack.cpp

```diff
@@ -0,0 +1,40 @@
+#include "atmosphere_prescribed_aerosol.hpp"
```

Comment:
I'm ok with this file, very temporarily though (if possible). As soon as the PR goes in, I would like to remove the hack, and fix it, if possible.

Reply:
Agreed